src/random_r.c

```diff
@@ -30,6 +30,7 @@
 
 #ifdef __MACH__
 # include "../include/features.h"
+#elif __unix__
```

Comment:
Well i could go deeper into checking for FreeBSD and NetBSD and OpenBSD, true. But i'm pretty sure they all run the same way. __LINUX__ should not classify as __unix__ tho :s